util/src/main/java/tc/oc/pgm/util/skin/Skin.java

```diff
@@ -0,0 +1,67 @@
+package tc.oc.pgm.util.skin;
+
+/** A self-contained skin */
+public class Skin {
+  // NOTE: ordinals must match bit positions in packet data
+  public static enum Part {
```

Comment:
Stated before, but let's not implement the skin parts, part. Just the data and signature should be ok.